### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by kpp**
Update nix and skim

### CODE REVIEW

**Comment by gwenn** [comment]
You should check `with-fuzzy` feature.
Github Actions don't.
As far as I know, `skim` > 0.7 fails.

**Comment by kpp** [comment]
> Github Actions don't.

Fixed.

> As far as I know, skim > 0.7 fails.

Fixed.

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -38,3 +38,24 @@ jobs:
...
+        override: true
+    - name: Run tests no-default-features
+      run: cargo test --workspace --all --doc --no-default-features
+    - name: Run tests only fuzzy
+      run: cargo test --workspace --all --doc --no-default-features --features with-fuzzy
+    - name: Run tests all features
+      run: cargo test --workspace --all --doc --features with-fuzzy,with-dirs
+
+    env:
+      RUST_BACKTRACE: 1
```
Comment (gwenn) [inline .github/workflows/rust.yml:61]: https://github.com/kkawakam/rustyline/pull/498/files
should be sufficient and faster

**Comment by kpp** [comment]
Would you please cut a new release?

**Comment by gwenn** [comment]
I will try to find some time this week.

**Comment by gwenn** [comment]
Version 8.0.0 released.

**Pull request merged**